.gitattributes

```diff
@@ -0,0 +1,2 @@
+/vendor/* linguist-vendored
+/vendor/* linguist-generated=true
```

Comment:
this is not the default?

Reply:
Seemingly not for some reason. When I was reviewing this PR, some of the `/vendor/**/*` files were expanded in the diff. Didn't feel like going down that rabbit hole so put in this as a fail safe.